library/native_protocol/repliers.cpp

```diff
@@ -40,7 +41,13 @@ int replier_base::reply_error(int errc, bool last) {
 static size_t calculate_body_size(const n2_serialized::chunks_t &chunks) {
 	size_t size = 0;
 	for (const auto &chunk : chunks) {
-		size += chunk.size();
+		auto visitor = make_lambda_visitor<size_t>(
```

Comment:
Not good that visitor object is created at each iteration (in such a lightweight loop).